test/parallel/test-https-agent-sockets-leak.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: can you please move this after the `common.hasCrypto`? Thank you.

Reply:
Sure. Thanks for reviewing.